.babelrc

```diff
@@ -1,8 +1,13 @@
 {
+<<<<<<< HEAD
```

Comment:
You didn't finish with the process of rebasing/resolving conflicts, please complete this process